SA/design_concept/content/poc/options_deployment_decision.typ

```diff
@@ -0,0 +1,9 @@
+= Deployment Decision
+
+We expected that VisualFP would primarily be used in classrooms. Such
+environments often do not allow for easy deployment and maintenance of
+application servers.
+
+Thus, the decision has been made to develop VisualFP as a desktop application.
+Given modern technologies such as @option-electron, this doesn't require us to
```

Comment:
I think using a `#link()` would be better here. @option-electron is shown as Section... which is a bit weird to read in this case.